scripts/ci/libraries/_initialization.sh

```diff
@@ -115,7 +115,7 @@ function initialization::initialize_base_variables() {
     export CURRENT_PYTHON_MAJOR_MINOR_VERSIONS
 
     # Currently supported versions of Postgres
-    CURRENT_POSTGRES_VERSIONS+=("9.6" "13")
+    CURRENT_POSTGRES_VERSIONS+=("10" "13")
```

Comment:
Should this also include 11 and 12, or should they be removed from `BREEZE.rst` instead? Right not it isn't consistent.

Reply:
This was deliberate decision.

Those are not consistent because we made a conscious decision that if CI works for "lowest" and "highest" version - it works for those between. 

We NEVER saw postgres related version problem. Not that I am aware of (as opposed to our Elephant in the room MySQL which keeps on appearing now and then). this was pure optimization for CI.

Reply:
Gotcha, cool, makes sense 👍